cvat/schema.yml

```diff
@@ -10157,14 +9959,6 @@ components:
         * `supervisor` - Supervisor
         * `maintainer` - Maintainer
         * `owner` - Owner
-    RqId:
```

Comment:
This renaming seems like a gratuitous compatibility break for me. IMO, we shouldn't do it.